lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -36,8 +36,10 @@ class Map
 module Fluent
   # fluentd output plugin for the Stackdriver Logging API
   class GoogleCloudOutput < BufferedOutput
-    # Constants for service names and resource types.
+    # Constants used by the plugin and its tests.
```

Comment:
Oops, this change is not part of the refactoring. It only makes sense after adding Metadata Agent support. I just reverted it.